build/build-image.sh

```diff
@@ -41,14 +41,37 @@ dir=$1
 image=$2
 
 docker build "$ROOT" \
```

Comment:
As discussed, we decided to not build e.g. `cortexlabs/python-predictor-gpu-slim:<version>`